.github/actions/latest-wrangler/action.yml

```diff
@@ -0,0 +1,20 @@
+name: "Github package 'latest' tag wrangler for containers"
+description: "Determines wether or not a given dbt container should be given a bare 'latest' tag (I.E. dbt-core:latest)"
+inputs:
+  package_name:
+    description: "Package to check (I.E. dbt-core, dbt-redshift, etc)"
+    required: true
+  new_version:
+    description: "Semver of the container being built (I.E. 1.0.4)"
+    required: true
+  gh_token: 
+    description: "Auth token for github (must have view packages scope)"
+    required: true
+outputs:
+  latest:
+    description: "Wether or not built container should be tagged latest (bool)"
+  minor_latest:
+    description: "Wether or not built container should be tagged minor.latest (bool)"
```

Comment:
General question.. I've never known what to use as return types for GHA & scripting things. This returns `True` | `False`, but does it make sense to return 0 | 1 or other bool representations?

Reply:
Ok, so this is something that took a while to sort out, but the TL;DR is that it doesn't matter.  GHA is all written in javascript, which types _everything_ dynamically so any variable you set  as an output operates in the same way.  

I've defaulted to using string representations by default because all values [coming from bash are untyped](https://tldp.org/LDP/abs/html/untyped.html) and treated as strings.  It's one less thing to have to remember. :)